mono/metadata/threadpool.c

```diff
@@ -288,10 +288,19 @@ try_invoke_perform_wait_callback (MonoObject** exc, MonoError *error)
 	HANDLE_FUNCTION_RETURN_VAL (res);
 }
 
-static gsize
+static void
 set_thread_name (MonoInternalThread *thread)
 {
-	return mono_thread_set_name_constant_ignore_error (thread, "Thread Pool Worker", MonoSetThreadNameFlag_Reset);
+	// Any thread can set any other thread name at any time.
```

Comment:
Given this duplicates knowledge from set_constant, merits a comment or a helper macro/function.

Reply:
I added a funny named flag to centralize the knowledge.